src/adapters/index.ts

```diff
@@ -1,2 +1,3 @@
-export * from './typeorm.adapter';
 export * from './object.adapter';
+export * from './strapi.adapter';
+export * from './typeorm.adapter';
```

Comment:
Should we avoid importing every adapters ?
I believe that a user of `typeorm` would still have to add `strapi` to its dependencies (I may be mistaking because I am no expert on javascript module loading here)

Reply:
Fear 😱 
I checked : typeorm is only in the devDependencies and not included when installed in your repo. It also does not crash if the dependency is missing even if the import is being made.